src/main/java/com/epam/lowcost/DAO/implementations/TicketDAOImpl.java

```diff
@@ -35,7 +35,7 @@ public TicketDAOImpl(DataSource dataSource, RowMapper<User> userRowMapper, RowMa
                         "JOIN  USERS ON TICKETS.userId=USERS.id " +
                         "JOIN  FLIGHTS ON TICKETS.flightId=FLIGHTS.id " +
                         "JOIN  PLANES ON FLIGHTS.planeId = PLANES.id " +
-                        "WHERE TICKETS.isDeleted=false and TICKETS.userId=%d", currentUserId);
+                        "WHERE TICKETS.isDeleted=false and TICKETS.userId=%d ORDER BY TICKETS.ID DESC", currentUserId);
```

Comment:
What is the reason for using sort by id ?

Reply:
the recent bought ticket should be on the top

Reply:
No. It is incorrect thought. `Id` is a simple identifier number, why some row has the highest `id` no matter. You can  add field: purchaseDate, for example.

Reply:
removed